docs/DeveloperGuide.md

```diff
@@ -103,11 +103,18 @@ The sequence diagram below illustrates the interactions within the `Logic` compo
 
 How the `Logic` component works:
 
-1. When `Logic` is called upon to execute a command, it is passed to an `TutorEaseParser` object which in turn creates a parser that matches the command (e.g., `DeleteContactCommandParser`) and uses it to parse the command.
-1. This results in a `Command` object (more precisely, an object of one of its subclasses e.g., `DeleteContactCommand`) which is executed by the `LogicManager`.
-1. The command can communicate with the `Model` when it is executed (e.g. to delete a person).<br>
+1. When `Logic` is called upon to execute a command, in this case `execute("contact delete 1)`, it is passed 
+   to an `TutorEaseParser` object which in turn creates a parser that matches the type of command. In this 
+   case, it is the `ContactCommandParser` as we are executing a command related to contacts. Commands 
+   related to lessons will use `LessonCommandParser`.
+2. The `ContactCommandParser` will create the specific `ContactCommandParser` corresponding to the 
+   execution. In this case, a `DeleteContactCommandParser` will be created to parse the command.
+3. This results in a `Command` object, specifically a `DeleteContactCommand` in this case, which is executed by the`LogicManager`.<br>
+   (Note that DeleteContactCommand` is a subclass of `ContactCommand` which is a subclass of `Command`)
+4. The command can communicate with the `Model` when it is executed, in this case, to delete a person.<br>
    Note that although this is shown as a single step in the diagram above (for simplicity), in the code it can take several interactions (between the command object and the `Model`) to achieve.
-1. The result of the command execution is encapsulated as a `CommandResult` object which is returned back from `Logic`.
+5. The result of the command execution is encapsulated as a `CommandResult` object which is returned back 
+   from `Logic`.
```

Comment:
Perhaps we can use 1. for all bullet points to follow the markdown convention.
I don't think a line break is needed as there is no line limit in markdown.

Reply:
I agree with what Norbert had mentioned. Otherwise, LGTM!

Reply:
@Chronoxy @NorbertLoh Fixed